package.json

```diff
@@ -1,148 +1,153 @@
+        "karma-sourcemap-loader": "^0.3.7",
+        "karma-webpack": "^3.0.5",
+        "lint-staged": "^5.0.0",
+        "lodash": "^4.17.5",
+        "mocha": "^4.0.1",
+        "mock-local-storage": "^1.0.2",
+        "mojito-rb-gen": "^0.0.1",
+        "node-noop": "^1.0.0",
+        "node-sass": "4.9.3",
+        "npm-run-all": "^4.1.5",
+        "optimize-css-assets-webpack-plugin": "^3.2.0",
+        "phantomjs-prebuilt": "^2.1.16",
+        "postcss-loader": "^2.0.9",
+        "postcss-sass": "^0.3.0",
+        "prettier": "^1.8.2",
+        "prettier-eslint-cli": "^4.4.2",
+        "raw-loader": "^0.5.1",
+        "react": "^16.7.0",
+        "react-dom": "^16.7.0",
+        "react-virtualized": "^9.13.0",
+        "sass-loader": "^6.0.6",
+        "sinon": "^7.2.3",
+        "sinon-chai": "3.3.0",
+        "string-replace-loader": "^1.3.0",
+        "style-loader": "^0.19.0",
+        "stylelint": "^8.3.1",
+        "stylelint-config-standard": "^18.0.0",
+        "stylelint-order": "^0.7.0",
+        "stylelint-scss": "^2.2.0",
+        "wait-on": "^3.2.0",
+        "webpack": "^3.10.0",
+        "webpack-bundle-analyzer": "^2.9.1",
+        "webpack-dev-server": "^2.11.3"
+    },
+    "engines": {
+        "node": ">=8.9.4",
+        "yarn": ">=1.10.0"
+    },
+    "scripts": {
+        "build": "yarn setup && yarn clean && yarn build:i18n && yarn lint && yarn build:dev",
+        "build:ci": "yarn setup && yarn webpack --progress --colors --config build/webpack.config.js",
+        "build:dev": "BABEL_ENV=dev NODE_ENV=dev yarn webpack --progress --colors --config build/webpack.config.js",
+        "build:i18n": "mojito-rb-gen -s src/i18n -o src/i18n/json -b en-US.properties",
+        "build:prod": "BABEL_ENV=production NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --progress --colors --config build/webpack.config.js",
+        "clean": "rm -rf dist && rm -rf reports/coverage && rm -rf src/i18n/json",
+        "cy:open": "yarn cy:wait; yarn cypress open",
+        "cy:run": "yarn cy:wait; yarn cypress run --spec test/integration/**/*.test.js",
+        "cy:wait": "wait-on http-get://localhost:8000",
+        "lint": "npm-run-all lint:*",
+        "lint:css": "NODE_ENV=dev yarn stylelint 'src/lib/**/*.scss'",
+        "lint:js": "NODE_ENV=dev yarn eslint src/lib",
+        "release": "yarn setup && yarn clean && yarn build:i18n && yarn lint && yarn test && yarn build:prod",
+        "release:major": "./build/release.sh -m",
+        "release:minor": "./build/release.sh -n",
+        "release:patch": "./build/release.sh -p",
+        "setup": "yarn install",
+        "start": "yarn setup && yarn build:dev --watch",
+        "start:dev": "yarn build:i18n && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev yarn webpack-dev-server --config build/webpack.config.js",
+        "test": "yarn setup && yarn clean && yarn build:i18n && NODE_ENV=test yarn karma start build/karma.conf.js",
```

Comment:
Is the addition of `yarn build:i18n` needed for the tests?

Reply:
Yes, unfortunately. The prior command had it but it was hidden away in the `ci` command, iirc.